.github/workflows/dapr.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     name: Build ${{ matrix.target_os }}_${{ matrix.target_arch }} binaries
     runs-on: ${{ matrix.os }}
     env:
-      GOVER: "^1.19.2"
+      GOVER: "1.19.2"
```

Comment:
If you pin a version, should at least be 1.19.13 which is the latest version. There have been security fixes too in-between: https://go.dev/doc/devel/release#go1.19.minor

But even better would be to change the setup-go action to read from the go.mod file: https://github.com/dapr/dapr/blob/5ac2f3d601dedc49913d7a848c49c1b8df660893/.github/workflows/dapr.yml#L138-L142

Reply:
Good point! I didn't know about this option; it's definitely much cleaner.